index.html

```diff
@@ -3857,6 +3857,73 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media type used by this specification relies on JSON-LD, JSON-LD
+processing is not required to be performed, since JSON-LD is JSON. Some
+scenarios where processing a <a>verifiable credential</a> or a <a>verifiable
+presentation</a> as JSON is desirable include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+Prior to verification when using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into systems that store or index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed for securing mechanisms that require an understanding of
+and processing of JSON-LD.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
```

Comment:
```suggestion
That is,  processing as vanilla JSON is allowed as long as the document being
consumed or produced is a <a>conforming document</a>. If vanilla JSON processing
is desired, an implementer is advised to follow the following rule:
```

Reply:
I have implemented @OR13's suggestion in 0e572793cefd18a51bdcb66ca439dba033846855.

We no longer use the word "pure" or "vanilla" and now just refer to "JSON" and "JSON-LD".